### PR TITLE
Update script to an nmp package for compatibility on with windows: mkdir -p --> mkdirp (#626).

### PR DESCRIPTION
**Pull request by markkulube**
Issue #626 

## Why is this PR for?
This script, defined in the root package.json, makes use of mkdir -p which is not working on Windows. We should replace it by the npm package mkdirp

Describe the reason of this PR or give a link towards the associated issue.
https://github.com/dubzzz/fast-check/issues/626

## In a nutshell

❌ New feature
✔️ Fix an issue
❌ Documentation improvement
❌ Other: *please explain*

(✔️: yes, ❌: no)

## Potential impacts

If you expect some noticeable impacts with your change please explain.

Here are some examples of impacts that might be reported into this section:
- Generated values impact,
- Shrink values impact,
- Performance impact,
- Typings impact

### CODE REVIEW

**Comment by markkulube** [comment]
@dubzzz I added mkdirp to devDependencies by running 'npm install mkdirp --save-dev' in the root directory. 

package.json was updated as per the most recent commit.

but i don't know why the Travis check keeps failing - any idea how to resolve this?

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "test:rollup-esm": "rollup --config test/rollup/esm/rollup.config.js && node test/rollup/esm/dist/main.js",
     "test:rollup-iife": "rollup --config test/rollup/iife/rollup.config.js && node test/rollup/iife/dist/main.js",
     "coverage": "cat ./coverage/lcov.info | coveralls",
-    "docs": "mkdir -p docs/2-API && docsify-init --editDir documentation && cp -R documentation/* docs/ && api-extractor run --local && generate-ts-docs markdown -i docs/2-API -o docs/2-API && cat README.md | sed 's/https:\\/\\/github.com\\/dubzzz\\/fast-check\\/blob\\/master\\/documentation//g' > docs/README.md && markdownbars -i docs/_sidebar.md -o docs/_sidebar.md",
+    "docs": "mkdirp docs/2-API && docsify-init --editDir documentation && cp -R documentation/* docs/ && api-extractor run --local && generate-ts-docs markdown -i docs/2-API -o docs/2-API && cat README.md | sed 's/https:\\/\\/github.com\\/dubzzz\\/fast-check\\/blob\\/master\\/documentation//g' > docs/README.md && markdownbars -i docs/_sidebar.md -o docs/_sidebar.md",
```
[the remark this answers is not in the document]
Comment (markkulube) [reply package.json:38]: @dubzzz I added mkdirp to devDependencies by running 'npm install mkdirp --save-dev' in the root directory.

package.json was updated as per the most recent commit.

but i don't know why the Travis check keeps failing - any idea how to resolve this?

[the remark this answers is not in the document]
Comment (dubzzz) [reply package.json:38]: Actually we are not using `npm` but `yarn` for the project

[the remark this answers is not in the document]
Comment (dubzzz) [reply package.json:38]: As you have not updated the `yarn.lock` with respect to the `package.json`, travis complains about it:

```
The command "eval yarn --frozen-lockfile " failed. Retrying, 2 of 3.
```

[the remark this answers is not in the document]
Comment (markkulube) [reply package.json:38]: @dubzzz should be good now, let me know if anything thanks!

**Comment by dubzzz** [comment]
Thanks a lot for the contribution

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dubzzz) [reply package.json:38]: Perfect 👌

**Pull request merged**